jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -75,6 +75,16 @@ properties:
         type: string
         fits_keyword: HGA_STOP
         blend_table: True
+      pwfseet:
+        title: Previous WFS exposure end time
+        type: number
+        fits_keyword: PWFSEET
+        blend_table: True
+      nwfsest:
+        title: Next WFS exposure start time
+        type: number
+        fits_keyword: NWFSEST
+        blend_table: True
```

Comment:
Since there's no limit on characters for names in the meta tree, would it make sense for `meta.pwfseet` and `meta.nwfsest` to be a bit more descriptive?  `prev_wfs_exp_end_time`?  or similar?

Reply:
Yes!

Reply:
I agree, but unfortunately SDP already added these to the Keyword Dictionary and they chose the short (same as the keyword) names, so we have to match. Someday when we're all done with the pipeline and we have nothing to do, I intend to go back and have SDP+CAL do a thorough scrub of the KWD to clean up and improve lots of things like this. Fortunately right now we have no use for these keywords in the Cal pipeline, so we don't need to worry about putting the cryptic names in our code.

Reply:
Wait, does SDP use our datamodel schemas?  In the JWST Keyword Dictionary, there's no reference to what we call it in the pipeline, at least available through the web interface.

I thought they only cared about what the FITS keyword was named.

Reply:
No, they don't use our data model schemas or even use the data model names that are in the KWD (at least not yet), but there's a comparator that gets run on two sets of schemas relatively frequently that points out all the differences, which then means someone (i.e. me) has to take time to investigate and explain them all the time. So for now it's just easier to keep them matched up. I have on several occasions had the KWD get changed to match us. But the week before a delivery deadline is not the time to make one of those requests.

Reply:
So the comparison is between their schemas and our schemas.  But if we're only comparing the FITS keyword, are we not free to make the name in `meta` that maps to the FITS keyword something a bit more descriptive?  We do that everywhere in our schemas.

Reply:
The KWD schemas have data model names mapped to keyword names, just like our schemas, because their schemas were created as a copy of ours years ago (or at least they used ours as a template). They add many more attributes to each keyword than we do, for things like sql_datatype, long description, units, example values, calculation to derive the keyword value, etc.